src/brain/issueLabelHandler/project.ts

```diff
@@ -0,0 +1,55 @@
+import { EmitterWebhookEvent } from '@octokit/webhooks';
+import * as Sentry from '@sentry/node';
+import { ISSUES_PROJECT_NODE_ID, PRODUCT_AREA_FIELD_ID } from '@/config';
+import { getIssueDetailsFromNodeId, getProductAreaFromProjectField } from './helpers';
+import { getClient } from '@api/github/getClient';
+import { ClientType } from '@/api/github/clientType';
+import { PRODUCT_AREA_LABEL_PREFIX } from '@/config';
+import { shouldSkip } from '@/brain/issueLabelHandler/helpers';
+
+function checkForIssuesProject(payload) {
+    return payload?.projects_v2_item?.project_node_id !== ISSUES_PROJECT_NODE_ID;
+}
+
+function checkForProductAreaField(payload) {
+    return payload?.changes?.field_value?.field_node_id !== PRODUCT_AREA_FIELD_ID;
+}
+
+function checkForProjectItemNodeId(payload) {
+    return payload?.projects_v2_item?.node_id == null || payload?.projects_v2_item?.content_node_id == null;
+}
+
+export async function syncLabelsWithProjectField({
+    id,
+    payload,
+    ...rest
+}: EmitterWebhookEvent<'projects_v2_item.edited'>) {
+    const tx = Sentry.startTransaction({
+        op: 'brain',
+        name: 'issueLabelHandler.syncLabelsWithProjectField',
+    });
+
+
+    const reasonsToDoNothing = [
+        checkForIssuesProject,
+        checkForProductAreaField,
+        checkForProjectItemNodeId
```

Comment:
Hrm, since we're going the `shouldSkip` route, can we update these names to make it clearer? I think maybe ... ?

`checkForIssuesProject` ⇒ `isNotTheRightProject`
`checkForProductAreaField` ⇒ `isNotTheRightProjectField`
`checkForProjectItemNodeId ` ⇒ `doesNotIncludeANodeID`

Reply:
Or maybe `isNotInAProjectWeCareAbout` and `isNotAProjectFieldWeCareAbout` to be parallel with `isNotInARepoWeCareAbout`?

Reply:
When we get to `Waiting for:` labels will we extend this handler to cover that case?

Reply:
> Or maybe isNotInAProjectWeCareAbout and isNotAProjectFieldWeCareAbout to be parallel with isNotInARepoWeCareAbout?

I like this

> When we get to Waiting for: labels will we extend this handler to cover that case?

Yes, we can extend the handler to apply for the `Status` field too